src/applications/lgy/coe/components/CoeDocumentList.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+const documents = [
+  {
+    id: 0,
+    title: 'a document',
+    type: 'PDF',
+    timestamp: 'January 12, 2021',
+  },
+  {
+    id: 1,
+    title: 'another document',
+    type: 'PDF',
+    timestamp: 'February 11, 2021',
+  },
+];
+
+export const CoeDocumentList = props => {
+  if (documents.length > 0) {
+    return (
+      <>
+        <h2 className="vads-u-margin-bottom--0">
+          You have letters about your COE request
+        </h2>
+        <p className="vads-u-border-color--gray-lighter vads-u-border-bottom--1px vads-u-margin--0 vads-u-padding-y--4">
+          We’ve emailed you notification letters about your COE request. Please
+          read these and follow the steps they outline. You may need to take
+          action before we can make a final decision.
+        </p>
+        {documents.map((document, key) => {
+          return (
+            <div
+              key={key}
+              className="vads-u-border-bottom--1px vads-u-border-color--gray-lighter vads-u-margin-top--4 vads-u-padding-bottom--4"
+            >
+              <h3 className="vads-u-font-family--serif vads-u-margin--0">
+                {document.title}
+              </h3>
+              <p className="vads-u-margin-y--1p5">
+                Date Sent: {document.timestamp}
+              </p>
+              <a className="vads-u-margin--0" href="#">
+                <i
+                  aria-hidden="true"
+                  className="fas fa-download vads-u-padding-right--1"
+                />
+                {document.title} {document.type}
+              </a>
+            </div>
+          );
+        })}
+      </>
+    );
+  }
+  if (props.onUploadPage) {
```

Comment:
Should this be `if (!props.onUploadPage) {`? This section should show up when the status is `pending` and there are no documents to download right?

Reply:
@jerekshoe this section is only supposed to show up if the user is on the pendind-upload page.